deployments/vpa.yml

```
@@ -0,0 +1,25 @@
+    containerPolicies:
+      - containerName: redis
+        minAllowed:
+          cpu: "100m"
+          memory: "128Mi"
+        maxAllowed:
+          cpu: "1"
+          memory: "1Gi"
+  selector:
+    matchLabels:
+      app: redis
+      component: redis
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)